refs/schemas/priv/RFC-PRIV.md

```diff
@@ -476,6 +479,59 @@ It may happen that two or more Retention Policies conflict, e.g. the same data m
+
+Once the System knows the Legal Bases for each [Privacy Scope](#privacy-scope), it relies on [Legal Base Events](#legal-base-event) and [Consents](#consents) to determine, for each user, the active Legal Bases. E.g. when a Data Subject starts using the System services (`SERVICE-START` type of event) the Privacy Scope associated with `CONTRACT` Legal Base becomes active.
+
+For more details, refer to [Expected Behavior of Implementing Systems](./expected-behavior.md).
+
+### Legal Base
+
+| Property | Expected cardinality | Expected values |
+| --------------- | ------ | -------------------- |
+| `legal-base` | 1-* | One or more [Legal Base Term(s)](#legal-bases)|
+| `scope` |  0-1 | a [Privacy Scope](#privacy-scope) covered by the given Legal Base(s). In absence of any indication the Event is considered to concern all the possible scope. |
+
+
+### Legal Base Event
+
+| Property | Expected cardinality | Expected values |
+| --------------- | ------ | -------------------- |
+| `data-subject` |  1-* | Data Subject concerned by the event, identified by one or more of their [Data Subject Identities](#decentralized-identity-of-data-subjects) each containing one `dsid` and one `dsid-schema`|
+| `event-type` | 1 | Any of the [Event Terms](#events) |
+| `legal-base` | 1-* | Any [Legal Base Term](#legal-bases) other than `CONSENT`|
+| `data-reference` | 0-* | optionally one or more references that uniquely identify the data that the event concerns (e.g. a legal case file reference, account ID, contract ID, a URL - For example, if the user canceled a particular subscription contract it is the contract ID) |
+| `date` | 1 | Date and Time of the Event given in JSON Schema [date-time](https://json-schema.org/draft/2020-12/json-schema-validation.html#rfc.section.7.3.1) format |
+| `scope` |  0-1 | a [Privacy Scope](#privacy-scope) gained or lost by this particular event. Specified only for special cases where a particular event affects only a part of the scope of the Legal Base. If not specified, the total Privacy Scope of the Legal bases is assumed. |
+
+
+Legal Base Events are a key element in evaluation of Retention Policies as well as for determining the validity of certain legal bases for data processing, especially `LEGITIMATE-INTEREST` or `CONTRACT` legal bases.
+
+Systems have the interest to register and exchange such information. For example, if a user canceled a subscription an Event of `event-type`:`SERVICE-END` should be created, with `data-reference` set to the ID of the subscription contract being canceled. This does not directly imply that the user's data should be deleted. The same user might have other ongoing contracts, or might have given consent for processing. The event is simply of the information that Privacy Compilers take into account when resolving retention policies and legal bases for data processing.
+
+
+### Consent
```

Comment:
I think Consent should not be an Event, but rather be attached to a Legal Base Event.

Reasoning about Events is simpler if all Events have the same structure and an associated PRIV object (particular Consent, Contract template, Privacy Request etc.)